staging/src/k8s.io/api/apps/v1beta1/types.go

```diff
@@ -56,6 +56,10 @@ type ScaleStatus struct {
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:prerelease-lifecycle-gen:introduced=1.6
+// +k8s:prerelease-lifecycle-gen:deprecated=1.8
+// +k8s:prerelease-lifecycle-gen:removed=1.22
```

Comment:
all apps/v1beta1 and apps/v1beta2 types are unserved as of 1.18 https://github.com/kubernetes/kubernetes/pull/85903

Reply:
> all apps/v1beta1 and apps/v1beta2 types are unserved as of 1.18 #85903

Look at me, being too generous